README.MD

```diff
@@ -69,8 +71,105 @@ benchmark = rxr.open_rasterio('benchmark_map_two_class_categorical.tif', mask_an
 
 ![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/metric_table.png)
 
-For more details on how to use this software, check out this [notebook
-tutorial](https://github.com/NOAA-OWP/gval/blob/main/notebooks/Tutorial.ipynb).
+### Continuous Example
+
+The same can be done for continuous rasters as shown below (in this case
```

Comment:
for rasters with continuous valued statistical data types

Reply:
Changed wording as requested.